ackhandler/interfaces.go

```diff
@@ -16,6 +16,7 @@ type SentPacketHandler interface {
 
 	SendingAllowed() bool
 	GetStopWaitingFrame(force bool) *wire.StopWaitingFrame
+	GetLowestPacketNotConfirmedAcked() protocol.PacketNumber
```

Comment:
@lucas-clemente: Any idea for a better name?

Reply:
SGTM.